game/assets/jak1/text/game_custom_text_en-US.json

```diff
@@ -205,7 +205,7 @@
   "1524": "ALL CUTSCENES",
   "1600": "INPUT OPTIONS",
   "1601": "SELECT CONTROLLER",
-  "1602": "ANALOG DEADZONE",
+  "1602": "STICK DEADZONE",
```

Comment:
thoughts on walking back this change considering these strings have likely already been translated?  I don't really think this was a major source of confusion.  I understand that technically the pressure sensitive buttons are analog buttons, but conventionally, most people don't refer to them as that.

Reply:
Do they not show up as 'outdated' in crowdin if we change them here?

Reply:
They would but im just saying that might be causing unnecessary work for several languages

Reply:
I can revert it, but keep in mind this adds a new string as well, so I'm not sure how much work it really adds. 2 strings vs 1.

Reply:
1 string is already translated by people that may never come back, that's the difference.